jetty-util/src/main/java/org/eclipse/jetty/util/thread/ShutdownThread.java

```diff
@@ -37,11 +36,11 @@
 public class ShutdownThread extends Thread
 {
     private static final Logger LOG = LoggerFactory.getLogger(ShutdownThread.class);
-    private static final ShutdownThread INSTANCE = new ShutdownThread();
+    private static final ShutdownThread _thread = new ShutdownThread();
```

Comment:
static final is all caps by checkstyle rules.

grandfathered ruleset is `__<name>` (two underscores).